00-HTML-CSS-basics/Ex1/expenses.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="author" content="Jacobo Andrés Dominguez"/>
+  <meta name="description" content="Exercise 1 | Doctypes and Metatags"/>
+  <meta name="keywords" content="Globant, Bootcamp, HTML5"/>
+  <meta name="viewport" content="width=device-width,initial-scale=1"/>
+  <title>Expenses</title>
+</head>
+<body>
+  <header>
+    <h1>Expenses</h1>
+  </header>
+  <table>
+    <tr>
```

Comment:
This is fine, you may need to know that is a good practice to add `thead` and `tbody` to the table structure. Anyway this is not needed to be applied.